common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp

```diff
@@ -184,12 +184,20 @@ class TrtCommon  // NOLINT
    */
   void setup();
 
+#if (NV_TENSORRT_MAJOR * 1000) + (NV_TENSORRT_MINOR * 100) + NV_TENSOR_PATCH >= 8500
+  void setupBindings(std::vector<void *> & bindings);
+#endif
+
   bool isInitialized();
 
   nvinfer1::Dims getBindingDimensions(const int32_t index) const;
   int32_t getNbBindings();
   bool setBindingDimensions(const int32_t index, const nvinfer1::Dims & dimensions) const;
+#if (NV_TENSORRT_MAJOR * 1000) + (NV_TENSORRT_MINOR * 100) + NV_TENSOR_PATCH >= 8500
+  bool enqueueV3(cudaStream_t stream);
+#elif (NV_TENSORRT_MAJOR * 1000) + (NV_TENSORRT_MINOR * 100) + NV_TENSOR_PATCH < 10000
```

Comment:
This way in current Autoware with TensorRT 8.6.1  `#elif` will not be reached and packages which use tensorrt_common will result with build fail. Please change if & elif to if & if. Both API wiil be available for TensorRT 8.5 < and TensorRT 10.0 >. We will refactor other nodes to use enqueueV3 later, now we want to merge this PR.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/9018/commits/c518d634be3ff1e0ef1fc9070899d31fe44c6e38

I can build current `yolox` on `trt` `8.6.1`.